#!/usr/bin/env python
PACKAGE = "sparse_to_dense"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("gradient_cutoff",    double_t,    0, "Cutoff value of depth pred gradient", 0.4,  0, 2)

exit(gen.generate(PACKAGE, "sparse_to_dense", "SparseToDense"))